ts/state/selectors/conversations.ts

```diff
@@ -114,15 +114,13 @@ export const _getLeftPaneLists = (
   conversations: Array<ConversationType>;
   archivedConversations: Array<ConversationType>;
 } => {
-  const values = Object.values(lookup);
-  const sorted = values.sort(comparator);
-
   const conversations: Array<ConversationType> = [];
   const archivedConversations: Array<ConversationType> = [];
 
-  const max = sorted.length;
+  const values = Object.values(lookup);
+  const max = values.length;
```

Comment:
I'm not convinced that the changes in this function have any impact on performance. Can you say more about the spirit of the changes here?

Reply:
The spirit is explained in the commit message, did you see this?

Reply:
(And what I didn't mention in the commit message: Not only is sorting two small arrays faster than one  large one, with the changes here we also don't need to sort the conversations for which `activeAt` is false.

Reply:
Ah yes, a nice speed-up if you have a lot of items without `activeAt`.

Let's get that memoizing updated, and then we can merge this. Thanks!